packages/lending/src/protocols/morphoblue/lending-protocol.ts

```diff
@@ -0,0 +1,229 @@
+      },
+      {
+        target: oracle,
+        callData: this.oracleIface.encodeFunctionData('price'),
+      },
+    ];
+
+    const { returnData } = await this.multicall3.callStatic.aggregate(calls, { blockTag: this.blockTag });
+
+    const { borrowShares, collateral } = this.morphoIface.decodeFunctionResult('position', returnData[0]);
+
+    const { totalSupplyAssets, totalSupplyShares, totalBorrowAssets, totalBorrowShares, lastUpdate, fee } =
+      this.morphoIface.decodeFunctionResult('market', returnData[1]);
+
+    const [_loanTokenPrice] = this.priceFeedIface.decodeFunctionResult('latestAnswer', returnData[2]);
+
+    // It corresponds to the price of 10**(collateral token decimals) assets of
+    // collateral token quoted in 10**(loan token decimals) assets of loan token
+    // with `36 + loan token decimals - collateral token decimals` decimals of precision.
+    const [normalizedCollateralLoanPrice] = this.oracleIface.decodeFunctionResult('price', returnData[3]);
+    const normalizedCollateralLoanDecimals = 36 + loanToken.decimals - collateralToken.decimals;
+
+    const loan = totalBorrowAssets.mul(borrowShares).div(totalBorrowShares);
+
+    const supplyBalance = common.toBigUnit(collateral, collateralToken.decimals);
+    const borrowBalance = common.toBigUnit(loan.toString(), loanToken.decimals);
+    const totalSupply = common.toBigUnit(totalSupplyAssets, collateralToken.decimals);
+    const totalBorrow = common.toBigUnit(totalBorrowAssets, loanToken.decimals);
+
+    const loanTokenPrice = common.toBigUnit(_loanTokenPrice, 8);
+    const collateralTokenPrice = common.toBigUnit(
+      _loanTokenPrice.mul(normalizedCollateralLoanPrice).div(BigNumber.from(10).pow(normalizedCollateralLoanDecimals)),
+      8
+    );
+
+    const loanValue = new BigNumberJS(borrowBalance).times(loanTokenPrice);
+    const collateralValue = new BigNumberJS(supplyBalance).times(collateralTokenPrice);
+    const ltv = loanValue.div(collateralValue).toFixed(5);
+
+    // Get Borrow Apy
+    const irmContract = Irm__factory.connect(irm, this.provider);
+    const marketParams: MarketParamsStruct = {
+      loanToken: loanToken.address,
+      collateralToken: collateralToken.address,
+      oracle,
+      irm,
+      lltv,
+    };
+    const market: MarketStruct = {
+      totalSupplyAssets,
+      totalSupplyShares,
+      totalBorrowAssets,
+      totalBorrowShares,
+      lastUpdate,
+      fee,
+    };
+    const borrowRateView = await irmContract.borrowRateView(marketParams, market, { blockTag: this.blockTag });
+    const borrowApy = (
+      Math.exp(new BigNumberJS(borrowRateView.toString()).div(1e18).times(SECONDS_PER_YEAR).toNumber()) - 1
+    ).toString();
+
+    const supplies: SupplyObject[] = [
+      {
+        token: collateralToken,
+        price: collateralTokenPrice,
+        balance: supplyBalance,
+        apy: '0',
+        usageAsCollateralEnabled: true,
+        ltv: ltv,
```

Comment:
```suggestion
        ltv,
```

Reply:
Morpho blue operates with a single loan and single collateral market, where the `LLTV` is equal to both the `ltv`(max LTV) and the `liquidationThreshold`.


Also, a note on naming: the `ltv` property here should ideally be renamed to `maxLTV`. We could consider addressing this after the current MR is completed.